docs/prod-host-setup.md

```diff
@@ -308,7 +308,7 @@ cat /sys/devices/system/cpu/vulnerabilities/spectre_v2
 
 The output should mention the following mitigations being in use:
 
-- `Enhanced IBRS` or `Retpolines`
+- `Enhanced IBRS`, `IBRS` or `Retpolines` (for pre-Skylake CPUs).
 - `IBPB` at least `conditional`
```

Comment:
I think it is a little unclear whether this is `(eIBRS && IBRS) || Retpolines` or `eIBRS || IBRS || Retpolines`.
```suggestion
The output should mention the following mitigations being in use:

- At least one of: `Enhanced IBRS`, `IBRS` or `Retpolines` *(`Retpolines` should be used for pre-Skylake CPUs)*.
- `IBPB` at least `conditional`


```

Reply:
how about this? 

- One of `Retpolines` (pre-Skylake CPU), `IBRS` (Skylake), or `Enhanced IBRS` (Cascade Lake and later)